kyo-prelude/shared/src/main/scala/kyo/Aspect.scala

```diff
@@ -1,35 +1,32 @@
 package kyo
 
-object Aspect:
-
-    private[kyo] val local = Local.init(Map.empty[Aspect[?, ?, ?], Cut[?, ?, ?]])
-
-    def init[A, B, S](using Frame): Aspect[A, B, S] =
-        init(new Cut[A, B, S]:
-            def apply[S2](v: A < S2)(f: A => B < S) =
-                v.map(f)
-        )
-
-    def init[A, B, S](default: Cut[A, B, S])(using Frame): Aspect[A, B, S] =
-        new Aspect[A, B, S](default)
-
-    def chain[A, B, S](head: Cut[A, B, S], tail: Seq[Cut[A, B, S]])(using Frame) =
-        tail.foldLeft(head)(_.andThen(_))
-end Aspect
-
 import Aspect.*
 
-abstract class Cut[A, B, S]:
```

Comment:
I've moved `Cut` to the companion object to avoid polluting the `kyo` package namespace. The companion object is also moved to after the class for consistency.